src/main/java/net/apnic/rdap/autnum/config/AutnumConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package net.apnic.rdap.autnum.config;
+
+import net.apnic.rdap.autnum.filters.AutnumRouteFilter;
+import net.apnic.rdap.directory.Directory;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AutnumConfiguration
+{
+    @Bean
+    @Autowired
+    public AutnumRouteFilter autnumRouteFilter(Directory directory)
```

Comment:
Would a better name for Directory be CompositeResourceLocator? (i.e. can It support the ResourceLocator interface?)